src/apis/debt_token_api.ts

```diff
@@ -63,7 +63,7 @@ export class DebtTokenAPI implements ERC721 {
     constructor(web3: Web3, contracts: ContractsAPI) {
         this.web3 = web3;
         this.contracts = contracts;
-        this.assert = new Assertions(web3, this.contracts);
+        this.assert = new Assertions(this.contracts);
```

Comment:
Assertions takes `web3` as one of its constructor arguments -- please fix.

Reply:
🤔

Atom was previously wanting me to remove this. Not sure why...

Fixed.